drivers/starburst/test/metabase/driver/starburst_test.clj

```diff
@@ -284,6 +284,57 @@
       (is (= (str "impersonate:true")
              (:clientInfo jdbc-spec))))))
 
+(defn prepared-statements-helper
```

Comment:
nit: Since we have this test that tests `prepared-optimized`, not sure we need the make optimized make command.

Reply:
This was added just in case should we suspect some issue with this flag (e.g. the standard Metabase tests using the prepared statements a slightly different way)